lstchain/reco/utils.py

```diff
@@ -649,12 +655,37 @@ def get_effective_time(events):
     t_eff: float
     t_elapsed: float
     """
-    
+
+    # time differences between the events in the table (which in general are
+    # NOT all triggered events):
+    time_diff = np.diff(events.dragon_time)
```

Comment:
Better to use `events["dragon_time"]` so that it is more general and can be used when we read the events as a QTable.

Reply:
why is this more general for this particular case?

Reply:
Accessing the column as an attribute is an extra feature for pandas.DataFrame, but when I use this function for my DL3 tool, where I access the DL2 file as an astropy.table object, I cannot use this method of accessing.

Reply:
but there is not problem to use it here, where we are not expecting this functionality to change, right?

Reply:
Done, thx!

Reply:
@rlopezcoto No, on its own there is no problem here. But as we need to use it while creating DL3 file, it has to be without using this method of accessing